src/app/panels/terminal.js

```diff
@@ -406,14 +399,16 @@ class Terminal {
     return self._view.el
 
     function change (event) {
-      handleAutoComplete(event)
+      if (self._components.autoCompletePopup.handleAutoComplete(
+        event,
+        self._view.input.innerText)) return
```

Comment:
")) return" should go to the new line here, aligned with if

Reply:
well `standard` doesn't seems to complain ;)

Reply:
but it do complain if `return` is put in a new line (cause then it needs `{}` )